charts/handbrake/0.0.1/test_values.yaml

```diff
@@ -0,0 +1,91 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5800            #Required for web GUI
+appAdditionalServicesEnabled: true
+appAdditionalServices:
+  vnc:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 5900
+
+
+# Path to your optical drive device in the container
+additionalVolumeMounts: []
+#  - name: opticaldrive
+#    mountPath: /dev/serial/by-id/sr0-
+
+# Path to your zwave device on the host
+additionalVolumes: []
+#  - name: opticaldrive
+#    hostPath:
+#      path: /dev/serial/by-id/sr0-
+
+# affinity:
+#   nodeAffinity:
+#     requiredDuringSchedulingIgnoredDuringExecution:
+#       nodeSelectorTerms:
+#       - matchExpressions:
+#         - key: app
+#           operator: In
+#           values:
+#           - opticaldrive-controller???  #zwave-controller
+
+
+
+#All values here are set as the docker defaults.
+env:
+# Permissions Settings
+  USER_ID: "1000"
+  GROUP_ID: "1000"
+  UMASK:    
+# General Settings
+  #TZ: Etc/UTC    #Do we need this?
+  KEEP_APP_RUNNING: "0"
+  CLEAN_TMP_DIR: "1"
+# GUI Settings
+  DISPLAY_WIDTH: "1280"
+  DISPLAY_HEIGHT: "768"
+  SECURE_CONNECTION: "0"
+  VNC_PASSWORD:
+# Automated Conversion Preset
+  AUTOMATED_CONVERSION_PRESET: 'General/Very Fast 1080p30'
+  AUTOMATED_CONVERSION_FORMAT: 'mp4'
+  AUTOMATED_CONVERSION_KEEP_SOURCE: "1"
+  AUTOMATED_CONVERSION_NON_VIDEO_FILE_ACTION: 'ignore'
+
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+    mountPath: /config
+
+  storage:
+    enabled: false
+    emptyDir: false
+    mountPath: /storage  # This can be read only
+
+  watch:      
+    enabled: false
+    emptyDir: false
+    mountPath: /watch
+
+  output:
+    enabled: false
+    emptyDir: false
+    mountPath: /output
+
+
+## Somehow must enable user to add as many watch folders
+## All folders must start with "watch" and an index after
```

Comment:
Even if we do not have endless random mountpoint creation, you can still create appVolumeMounts for things like config, storage and watch

Reply:
I can't really find any info on what is different between them, or if they work in addition to each other.
Is there any source i can read about them?  Or if you can say with a few words? Thanks

Reply:
Between what? 
AppVolumeMounts is one thing, not multiple, so what do you mean with "between them"?

Reply:
sorry meant persistence and appvolumemounts, i've seen both in already deployed charts. And usually same path is on both persistence and appvolumemounts.
Should i use only AppVolMounts?